barretenberg/bootstrap_cache.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
very nice

Reply:
will need to add a check in ./barretenberg/bootstrap.sh to trigger this script if cache is set, ./bootstrap.sh fast will not run this by default?

Reply:
Ah added this just for docker_fast.sh for now